tools/capture_har/capturer/speed_index.js

```diff
@@ -0,0 +1,34 @@
+const fs = require("fs")
+
+const utils = require('../../capture_har/utils')
+const pwmetrics = require('pwmetrics')
+
+const speedIndex = async (url, userDataDir, currentRunId) => {
+    const speedIndexCmd = [];
+    const tempOutPath = 'output-'+currentRunId+".json"; // pwmetrics will output to here. we will read it and send back if user did not want a file. 
+    if (userDataDir == '') {
+      userDataDir = '/mnt/share/speed_index/chrome-user-data-'+currentRunId;
```

Comment:
Why should we store the chrome user data in the mount folder if no directory was passed in? Shouldn't it use the default behavior?